spark-testing/dcos_utils.py

```diff
@@ -0,0 +1,19 @@
+import sdk_cmd
+
+
+def get_task_container_id(task):
+    for status in task['statuses']:
+        if status['state'] == "TASK_RUNNING":
+            return status['container_status']['container_id']['value']
+
+    return None
+
+
+def agent_ssh(host_ip, cmd):
```

Comment:
Originally, this function appeared to avoid `shakedown` ssh issues without performing the big move to the latest `dcos-commons`. The move is planned in the scope of [DCOS-49680](https://jira.mesosphere.com/browse/DCOS-49680) so this module will most probably go away.